code/modules/mob/living/blood.dm

```diff
@@ -16,22 +16,30 @@
 			var/spare_blood = blood_volume - ((BLOOD_VOLUME_MAXIMUM + BLOOD_VOLUME_NORMAL) / 2) //Knock you to the midpoint between max and normal to not spam.
 			if(drip(spare_blood))
 				var/bleed_range = 0
+				var/lethal = FALSE
 				switch(spare_blood)
 					if(0 to 30) //20 is the functional minimum due to midpoint calc
 						to_chat(src, span_notice("Some spare blood leaks out of your nose."))
 					if(30 to 100)
 						to_chat(src, span_notice("Spare blood gushes out of your ears and mouth. Must've had too much."))
 						bleed_range = 1
-					if(100 to INFINITY)
+					if(100 to 300)
 						visible_message(span_notice("Several jets of blood open up across [src]'s body and paint the surroundings red. How'd [p_they()] do that?"), \
 							span_notice("Several jets of blood open up across your body and paint your surroundings red. You feel like you aren't under as much pressure any more."))
 						bleed_range = 3
+					if(300 to INFINITY)
```

Comment:
spare_blood isn't the percent blood you have, it's the total units you have over 580.

Reply:
Oh my mistake, I thought it was in %, I'll change it to the appropriate number